codeview/src/types/primitive.rs

```diff
@@ -334,6 +335,7 @@ primitives! {
     (0x0679, T_64PUINT16, "64 bit pointer to 128 bit unsigned int"),
     (0x067a, T_64PCHAR16, "char16 *"),
     (0x067b, T_64PCHAR32, "char32 *"),
+    (0x067c, T_64PCHAR8, "char8 *"),
```

Comment:
I double-checked that I'm not accidentally shadowing the new idents (an intentional duplicate yields a compilation error).